CHANGELOG.md

```diff
@@ -1,8 +1,9 @@
 # Changelog
 
-## Changes
+## [0.20.0] - 2020-08-03
```

Comment:
Could you also add the link to the changelog from this version to the previous one at the end of this file please ?

Reply:
Of course, my bad!